### ISSUE TITLE
Document fetch API particular behavior on the server

### ISSUE THREAD

**Issue by MoustaphaDev** [withastro/docs#3251]
On the server `fetch` needs to be passed the full url of the resource to retrieve

Wrong way
```astro
---
const res = await fetch("/api") // this will raise an error
---
```

Right way
```astro
---
const res = await fetch("http://localhost:3000/api")
---
```

**Comment by nienow** [comment]
The following works and could be added as an example:
```
---
const res = await fetch(Astro.url + "/api");
---
```

**Comment by sasoria** [comment]
This was solved in #3677 so I think it can be closed.

**Comment by delucis** [comment]
Thanks for addressing this @sasoria!
